api/src/main/java/org/pmiops/workbench/cohortbuilder/querybuilder/DemoQueryBuilder.java

```diff
@@ -85,6 +86,10 @@ public QueryJobConfiguration buildQueryJobConfig(QueryParameters parameters) {
                     throw new IllegalArgumentException("Dec must provide a value of: " + DECEASED);
                 }
 
+            } else if (key.equals(DEMOTYPE.ETH.name())) {
+                final String namedParameter = key.toLowerCase() + getUniqueNamedParameterPostfix();
+                queryParts.add(DEMO_ETH.replace("${eth}", "@" + namedParameter));
+                queryParams.put(namedParameter, QueryParameterValue.array(paramMap.get(key).stream().toArray(Long[]::new), Long.class));
```

Comment:
Perhaps not for this PR, but perhaps for a future refactoring, this series of if-else-ifs could be a switch on the `DEMOTYPE` enum. I think that would be a lot cleaner.